staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3468,12 +3467,12 @@ func (c *Cloud) findELBSubnets(internalELB bool) ([]string, error) {
 		// If we have two subnets for the same AZ we arbitrarily choose the one that is first lexicographically.
 		// TODO: Should this be an error.
 		if strings.Compare(*existing.SubnetId, *subnet.SubnetId) > 0 {
-			klog.Warningf("Found multiple subnets in AZ %q; choosing %q between subnets %q and %q", az, *subnet.SubnetId, *existing.SubnetId, *subnet.SubnetId)
+			klog.InfoS("Found multiple subnets in AZ", " AZ ",az, "choosingSubnetId", *subnet.SubnetId, "existingSubnetId", *existing.SubnetId,  "subnetId2", *subnet.SubnetId)
 			subnetsByAZ[az] = subnet
 			continue
 		}
 
-		klog.Warningf("Found multiple subnets in AZ %q; choosing %q between subnets %q and %q", az, *existing.SubnetId, *existing.SubnetId, *subnet.SubnetId)
+		klog.InfoS("Found multiple subnets in AZ", " AZ ",az, "choosingSubnetId", *existing.SubnetId,"existingSubnetId", *existing.SubnetId,  "subnetId2",*subnet.SubnetId)
```

Comment:
`Found multiple subnets in AZ.` -> `Found multiple subnets in AZ. Choosing one.`
`choosingSubnetId` -> `chosenSubnetId`
`existingSubnetId` -> `firstSubnet`
`subnetId2` -> `secondSubnet`